uninstall.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Plugin uninstall handler.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+use Google\Web_Stories\Story_Post_Type;
+use Google\Web_Stories\Template_Post_Type;
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	return;
+}
+
+/**
+ * Filters whether data should be erased when uninstalling the plugin.
+ *
+ * @param bool $erase Whether to erase data. Default false.
+ */
+$erase = (bool) apply_filters( 'web_stories_erase_data_on_uninstall', false );
```

Comment:
When will this code ever fire?

Reply:
At the moment, never. Unless someone does `add_filter( 'web_stories_erase_data_on_uninstall', '__return_true' ); ` somewhere.

Basically like a feature flag.

Once we have a UI option for this, this will change.

Reply:
Should this default to true and allow for an opt out? Otherwise, this will never fire.

Reply:
At the moment, no. Think of it as a feature flag.

We will change this once there is a settings page.

For now, this enables us to develop this until it's ready, without breaking anything.

Reply:
Am I correct in understanding there is still no UI to enable this? If so, is data currently never removed? We're following the model you've put forth here in https://github.com/ampproject/amp-wp/pull/6422 but we don't have this feature flag to prevent erasure of the data.

Reply:
@westonruter Looks like that's the case, yes. We've never followed up on this, even though we now have a settings page.

Interestingly, plugins like WooCommerce or Jetpack don't have a UI option for this and instead require setting a constant in `wp-config.php` to enable data removal.

I just opened #8453 to further discuss this.